src/test/moment/duration.js

```diff
@@ -148,6 +148,14 @@ test('instantiation from another duration', function (assert) {
     assert.deepEqual(moment.duration(modified), modified, 'cloning modified duration works');
 });
 
+test('explicit cloning', function (assert) {
+    var durationA = moment.duration(5, 'milliseconds');
+    var durationB = durationA.clone();
+    durationA.add(5, 'milliseconds');
+    assert.equal(durationB.milliseconds(), 5, 'Calling duration.clone() on a duration will create a clone');
+    assert.equal(durationA.milliseconds(), 10, 'Calling duration.clone() on a duration will create a clone');
```

Comment:
It would make more sense to test that `durationA.milliseconds()` does not equal `durationB.milliseconds()`. That would test that the object had in fact been cloned more explicitly